quickstart.sh

```diff
@@ -187,6 +187,7 @@ curl -S -X PUT -H "Content-Type: application/json" -d '{"name":"brand"}' http://
 
 if $offline_lab; then
   echo -e "${MAJOR}Setting up Quepid${RESET}"
+  sleep 5
```

Comment:
is this needed?    We have a couple of places we have polling where we are waiting for things.